benchmarks/nsinker/nsinker_2d.prm

```diff
@@ -0,0 +1,86 @@
+# This is the "NSinker" benchmark defined in May et al. (2015,G3) in the
```

Comment:
change this comment to just say that this is a 2d version of the originally 3d problem

Reply:
done.